lib/partitions_validator_utils.pm

```diff
@@ -24,4 +30,64 @@ sub validate_partition_table {
     assert_equals($args->{table_type}, $table_type, "Partition table type does not correspond to the expected one.");
 }
 
+sub validate_partition_table {
+    my ($args) = @_;
+    return if check_var('BACKEND', 's390x');                                       # fdisk output does not show partition table for dasd
+    record_info("Check $args->{table_type}", "Verify that partition table type is $args->{table_type}");
+    my $table_type = (split(/\"/, script_output("blkid $args->{device}")))[-1];
+    assert_equals($args->{table_type}, $table_type, "Partition table type does not correspond to the expected one.");
+}
+
+sub validate_partition_creation {
+    my ($args) = @_;
+    my @lsblk_output = split(/\n/, script_output("lsblk -n "));
+    record_info("Check $args->{mount_point}", "Verify that partition $args->{mount_point} was created.");
+    my $check = 0;
```

Comment:
I normally declare it without initialize it to 0, as it is `undef` anyway, but up to you.